tox.ini

```diff
@@ -82,6 +86,8 @@ commands =
          ext/opentelemetry-ext-azure-monitor/examples/ \
          ext/opentelemetry-ext-azure-monitor/src/ \
          ext/opentelemetry-ext-azure-monitor/tests/ \
+         ext/opentelemetry-ext-azure-monitor/src/ \
```

Comment:
```suggestion
         ext/opentelemetry-ext-jaeger/src/ \
```

Reply:
It explains a lot of things. Thanks for catching this up!